uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/MediaPopupElementView.qml

```diff
@@ -0,0 +1,165 @@
+
+import Esri.ArcGISRuntime.Toolkit.Controller
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+
+ColumnLayout {
+    id: mediaPopupElementView
+    height: childrenRect.height
+    clip: true
+    focus: true
+    Layout.fillWidth: true
+    spacing: layoutSpacing
+
+    property real mediaMargin: 10
+    property real imageTextMargin: 5
+    property real layoutSpacing: 0
+
+    /*!
+      \qmlproperty MediaPopupElementView controller
+      \brief The Controller handles reading from the MediaPopupElement.
+
+      \sa Esri::ArcGISRuntime::Toolkit::MediaPopupElementViewController
+    */
+    property var controller: null
+
+    MenuSeparator {
+        Layout.fillWidth: true
+        Layout.leftMargin: mediaMargin
+        Layout.rightMargin: mediaMargin
+    }
+
+    ColumnLayout {
+        clip: true
+        focus: true
+        spacing: layoutSpacing
+
+        Label {
+            text: controller.title !== "" ? controller.title : "Media"
+            wrapMode: Text.WordWrap
+            font.pixelSize: 20
+            font.weight: Font.Black
+
+            Layout.fillWidth: true
+            Layout.leftMargin: mediaMargin
+            Layout.rightMargin: mediaMargin
+        }
+        Label {
+            text: controller.description
+            visible: controller.description !== ""
+            wrapMode: Text.WordWrap
+
+            Layout.fillWidth: true
+            Layout.leftMargin: mediaMargin
+            Layout.rightMargin: mediaMargin
+        }
+    }
+
+    MenuSeparator {
+        Layout.fillWidth: true
+        Layout.leftMargin: mediaMargin
+        Layout.rightMargin: mediaMargin
+    }
+
+    ListView {
+        orientation: ListView.Horizontal
+        clip: true
+        focus: true
+        height: 170
+        spacing: mediaMargin
+        model: controller.popupMediaItems
+
+        Layout.fillWidth: true
+        Layout.leftMargin: mediaMargin
+        Layout.rightMargin: mediaMargin
+
+        delegate: Item {
+            height: 170
+            width: 220
+
+            Image {
+                id: imageMediaPreview
+                source: model.sourceUrl
+                fillMode: Image.PreserveAspectCrop
+                asynchronous: true
+                cache: true
+
+                anchors.fill: parent
+                Layout.leftMargin: mediaMargin
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        Qt.openUrlExternally(model.linkUrl);
+                    }
+                }
+            }
+
+            // Overlay text
+            Rectangle {
+                visible: model.title !== ""
+                height: overlayTextLayout.height
+                color: "white"
+                opacity: 0.7
+
+                anchors.bottom: parent.bottom
+                anchors.left: parent.left
+                anchors.right: parent.right
```

Comment:
typically prefer to group these

Reply:
I tried to do this for the Layout properties and remembered i couldn't and in my dismay, forgot about anchors.